index.html

```diff
@@ -9,24 +9,30 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <div class= "container-book">
-    <h1 class= "title">
-      <strong>Título del libro</strong> 
-      <p class= "genre"> (género)</p>
-    </h1>
-    <div class= "container-details">
-        <h2 class="title">
-        <strong>Autor del libro:</strong> 
-        <p class= "detail"> Nombre del autor del libro</p> 
-        </h2>
-        <h2 class="title">
-        <strong>Editorial:</strong>
-        <p class= "detail"> Nombre de la editorial</p>
-        </h2>
-        <h2 class="title">
-        <strong>Año de publicación:</strong>
-        <p class= "detail"> Año de publicación</p>
-        </h2>
+  <div class= "cover-text-container">
+    <img class= "cover-page" 
+    src= "Images/book-cover.png" 
+    alt= "Wolox books cover page"
+    >
+    <div class= "container-book">
+      <h1 class="book-title">
+        Título del libro
+        <p class= "book-genre"> (género)</p>
```

Comment:
Regarding a previous suggestion, it is possible to put the paragraph outside the heading. Your element are being positioned one below the other it's to be expected because of flex-column.
You can move the closing tag of  `<div class="book-title">` and only wrap both the h1 and p